packages/contracts/src/interfaces/IFluidLocker.sol

```diff
@@ -226,6 +335,12 @@ interface IFluidLocker {
      */
     function getStakedBalance() external view returns (uint256 sBalance);
 
+    /**
+     * @notice Returns this Lockers' Uniswap V3 Liquidity balance in the SUP/ETH pool
+     * @return lBalance Uniswap V3 Liquidity balance in the SUP/ETH pool
+     */
+    function getLiquidityBalance() external view returns (uint256 lBalance);
```

Comment:
I think something like `getTotalLiquidity()` returning an uint128 would be more consistent with `getPositionLiquidity()`. The term `balance` should be used either for both or for none.
Also please add to the natspec that this is the sum of liquidity in all the positions owned by the locker.